pkg/grafana/types.go

```diff
@@ -4,3 +4,11 @@ type Organization struct {
 	ID   int64
 	Name string
 }
+
+type Datasource struct {
+	ID     int64
+	Name   string
+	Type   string
+	Url    string
```

Comment:
```suggestion
	URL    string
```

as this is an acronym